package/redhat/smartdns.spec

```diff
@@ -0,0 +1,63 @@
+Name:           smartdns
+Version:        31
```

Comment:
Change the version number to a variable may better.

Reply:
there is a problem automatic detect the version in the repo
can we place a file in the root ,eg version.txt ?

Reply:
can we add this feature for the moment?